### ISSUE TITLE
add signin and signup to route config

### ISSUE THREAD

**Issue by nadunindunil** [akveo/blur-admin#160]
Hi,

first of all I must say blur admin is awesome!

my problem is why don't we add sign in,404 and sign up into route config rather than having those as static routes. Since whole application is a single page application having those views in the same flow would be nice.

thank you.

**Comment by dkarchmer** [comment]
+1

**Comment by vazh** [comment]
you can add it by yourself, you only need to edit a few files a little bit.

**Comment by trinvh2** [comment]
+1

**Comment by nadunindunil** [comment]
@vazh yes I knew that. I was suggesting adding those routes to the master blur admin project.

**Comment by nadunindunil** [comment]
if anyone interested here is the routes changed fork for login of blur admin - https://github.com/nadunindunil/blur-admin
